packages/compass-collection/src/components/collection-tab.tsx

```diff
@@ -118,10 +118,10 @@ function WithErrorBoundary({
 function useCollectionTabs(props: CollectionMetadata) {
   const pluginTabs = useCollectionSubTabs();
   const connectionInfoRef = useConnectionInfoRef();
-  const isGlobalWritesSupported = useConnectionSupports(
-    connectionInfoRef.current.id,
-    'globalWrites'
-  );
+  const isGlobalWritesSupported =
+    useConnectionSupports(connectionInfoRef.current.id, 'globalWrites') &&
+    !props.isReadonly &&
+    !['config', 'local', 'admin'].includes(props.namespace.split('.')[0]);
```

Comment:
We use `mongodb-ns` package for this. You'd want to check the `specialish` prop on the parsed namespace here

Reply:
Done!